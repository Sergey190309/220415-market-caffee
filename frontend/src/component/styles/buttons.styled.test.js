import React from 'react'
import renderer from 'react-test-renderer'

import { Button } from './buttons.styled'

describe('Button styled', () => {
  test('Button', () => {
    const tree = renderer.create(<Button />).toJSON()
    // console.log('buttons.styled ->', tree)

    expect(tree).toMatchSnapshot();
    expect(tree).toHaveStyleRule(
      'cursor', 'pointer'
    )
    expect(tree).toHaveStyleRule(
      'color', 'white', {
        modifier: ':hover'
      },
      'background-color', 'palevioletred', {
        modifier: ':hover'
      }
    )
  });
});