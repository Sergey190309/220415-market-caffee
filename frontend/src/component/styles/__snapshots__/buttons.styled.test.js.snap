// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button styled Button 1`] = `
<button
  className="sc-bczRLJ etCJzT"
/>
`;
